spring-cloud-vault-config-databases/src/main/java/org/springframework/cloud/vault/config/databases/VaultConfigDatabaseBootstrapConfiguration.java

```diff
@@ -38,13 +44,30 @@
  * @author Per Abich
  * @author Sebastien Nahelou
  * @author Francis Hitchens
+ * @author Quintin Beukes
  */
 @Configuration(proxyBeanMethods = false)
 @EnableConfigurationProperties({ VaultMySqlProperties.class, VaultPostgreSqlProperties.class,
 		VaultCassandraProperties.class, VaultCouchbaseProperties.class, VaultMongoProperties.class,
-		VaultElasticsearchProperties.class, VaultDatabaseProperties.class })
+		VaultElasticsearchProperties.class, VaultMultipleDatabaseProperties.class, VaultDatabaseProperties.class })
+@Order(Ordered.LOWEST_PRECEDENCE - 15)
```

Comment:
I changed the order to ensure the @PostConstruct runs before VaultBootstrapPropertySourceConfiguration.